syntax = "proto3";

import "google/protobuf/any.proto";

package opentera.protobuf;

message TeraEvent {
    message Header {
        uint32 version = 1;
        double time = 2;
	string topic = 3;
    }

    Header header = 1;
    repeated google.protobuf.Any events = 2;
}
